src/net.rs

```diff
@@ -218,7 +233,12 @@ impl NetworkAcceptor for HttpAcceptor {
     fn accept(&mut self) -> IoResult<HttpStream> {
         Ok(match *self {
             HttpAcceptor::Http(ref mut tcp, _) => HttpStream::Http(try!(tcp.accept())),
-            HttpAcceptor::Https(ref mut _tcp, _) => unimplemented!(),
+            HttpAcceptor::Https(ref mut _tcp, _, ref ssl_context) => {
```

Comment:
The `_tcp` before was to remove the unused warnings. Since it's used, the `_` can be removed.